src/applications/disability-benefits/686/config/form.js

```diff
@@ -269,41 +277,16 @@ function createLocationUISchemaForKey(
 ) {
   return {
     'ui:title': title,
-    countryDropdown: {
-      'ui:title': 'Country',
+    state: {
+      'ui:title': 'State (or country if outside the USA',
       'ui:required': isRequiredCallback,
-    },
-    countryText: {
-      'ui:title': 'Enter Country',
-      'ui:required': (formData, index) =>
-        isInternationalAddressText(
-          get(`${insertRealIndexInKey(key, index)}`, formData),
-        ),
-      'ui:options': {
-        hideIf: (formData, index) =>
-          isNotInternationalAddressText(
-            get(`${insertRealIndexInKey(key, index)}`, formData),
-          ),
-      },
+      
     },
     city: {
-      'ui:title': 'City',
-      'ui:required': (formData, index) =>
-        isUSAAddress(get(`${insertRealIndexInKey(key, index)}`, formData)),
-      'ui:options': {
-        hideIf: (formData, index) =>
-          !isUSAAddress(get(`${insertRealIndexInKey(key, index)}`, formData)),
-      },
-    },
-    state: {
-      'ui:title': 'State',
-      'ui:required': (formData, index) =>
-        isUSAAddress(get(`${insertRealIndexInKey(key, index)}`, formData)),
-      'ui:options': {
-        hideIf: (formData, index) =>
-          !isUSAAddress(get(`${insertRealIndexInKey(key, index)}`, formData)),
-      },
+      'ui:title': 'City or county', 
+      'ui:required': isRequiredCallback,
```

Comment:
Can this property be removed if we're always asking for city and state (or country) moving forward?